### ISSUE TITLE
Investigate how to reduce Ketcher bundle size

### ISSUE THREAD

**Issue by rrodionov91** [epam/ketcher#4958]
**Background**
Current js bundle size of Ketcher is 6.5MB. It leads to long initial ketcher loading time.

**Solution**
It needs to run bundle analyser and investigate options to reduce bundle size. Result of this ticket should be a list of other tickets related to concrete actions.

**Comment by AlexeyGirin** [comment]
Test cases are needed for that feature

**Comment by rrodionov91** [comment]
Report from bundle analyzer:
[report.html.zip](https://github.com/user-attachments/files/16481076/report.html.zip)

**Comment by rrodionov91** [comment]
ketcher main bundle - 7.1MB

- ketcher-core - 4.2MB
  -  monomers library - 2.8MB(66%)
 

- ketcher-react - 1.9MB

  - templates library - 530KB(27%)

- ketcher-macromolecules - 1MB 

- ketcher-standalone - 200KB

miew chunk - 1.2MB (loads on demand)
ketcher-standalone with full indigo - 260KB (loads in background after indigo without render module initialisation)


—- conclusion —-
- 47% of main bundle is occupied by the library. We need to split it to separate file and load in parallel with main bundle.
- dependencies (node_modules) occupied ~15% of bundle size. We can split it into separate chunk.

**Comment by AlexeyGirin** [comment]
No verification required
